library/alloc/src/string.rs

```diff
@@ -2530,12 +2531,51 @@ macro_rules! impl_eq {
 
 impl_eq! { String, str }
 impl_eq! { String, &'a str }
+impl_eq! { String, &String }
+impl_eq! { &String, str }
 #[cfg(not(no_global_oom_handling))]
 impl_eq! { Cow<'a, str>, str }
 #[cfg(not(no_global_oom_handling))]
 impl_eq! { Cow<'a, str>, &'b str }
 #[cfg(not(no_global_oom_handling))]
 impl_eq! { Cow<'a, str>, String }
+#[cfg(not(no_global_oom_handling))]
+impl_eq! { Cow<'a, str>, &String }
+
+macro_rules! impl_ord {
+    ($lhs:ty, $rhs: ty) => {
+        #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Looks like some of the stability attributes were copy-pasted.

Reply:
Fixed.